src/openforms/forms/models/form.py

```diff
@@ -408,6 +409,13 @@ def copy(self):
             variable.form = copy
             variable.save()
 
+        try:
+            self.confirmation_email_template.pk = None
+            self.confirmation_email_template.form = copy
+            self.confirmation_email_template.save()
+        except ObjectDoesNotExist:
```

Comment:
Generally it's best practice to catch the model-specific subclass of this exception:

```py
from openforms.emails.models import ConfirmationEmailTemplate
```

and then:

```suggestion
        except ConfirmationEmailTemplate.DoesNotExist:
```

Reply:
The avoid circular imports I never import models from other packages in `models.py`. Of course I can place the import at function level but not very fond of the either.

Reply:
A local import is okay here - though you are right in that it's a bit of a code smell 

IMO this model should either just be model fields of the Form model or be moved into the forms app, but that's a topic for another improvement 😅

Reply:
Pushed with a local import and decided to use `with suppress` instead of the `try/except`.